website/src/docs/guides/backups.md

```diff
@@ -54,16 +71,39 @@ This way, you can recover from catastrophic failures.
 ![Automatic Backupse](/docs/guides/backups/automatic_backups.dark.webp#dark =414x402)
 :::
 
-## Forks Information
+### Sync Backups with External Cloud Services
+
+Cross Device Sync in Tachiyomi is not planned in the future, but users can use 
+[Autosync for Google Drive](https://play.google.com/store/apps/details?id=com.ttxapps.drivesync)
+in order to sync backup files to Google Drive automatically with the following steps:
+
+1. Install the app from the link above
+
+2. Enable [Automatic Backups](/docs/guides/backups#enable-automatic-backups) and set it to your desired frequency and storage location.
+
+3. Download the latest backup from Google Drive and restore to whatever device you have
+
+Users who are familiar with [FolderSync](https://play.google.com/store/apps/details?id=dk.tacit.android.foldersync.lite)
+or [Tasker](https://play.google.com/store/apps/details?id=net.dinglisch.android.taskerm) can setup auto sync of their backups similarly
+
+## Additional Information for Forks
+
+::: warning
+This section explores some extra details regarding [Official Forks](/forks/).
+:::
+
+All [Official Tachiyomi Forks](/forks/) support the `proto.gz` format to backup/restore your library
 
-Some forks of **Tachiyomi** have specific limitations regarding backup restoration:
+In addition, some forks have specific limitations regarding backup restoration:
 
 - [Neko](/forks/Neko/) can only restore [MangaDex](/extensions/#all.mangadex) entries in a backup.
   > Entries from other sources will not transfer.
-- Some forks, such as [TachiyomiAZ](/forks/TachiyomiAZ/) and [TachiyomiSY](/forks/TachiyomiSY/), have built-in extensions.
-  > Example: [E-Hentai](/extensions/#all.ehentai), [nHentai](/extensions/#all.nhentai), and **Hitomi.la**.
-- Restoring a backup made by these forks to standard **Tachiyomi** will not restore titles from such extensions.
-  > While it is possible to restore titles to these forks, backing them up out of these forks is not possible.
-- Some forks only support restoring legacy backups and haven't implemented the changes needed for new backup formats yet.
+  > Migrate everything to MangaDex if you require to.
+- All forks have fork-specific settings and changes that might be saved in Backups.
+  Such settings are not restored in original Tachiyomi and will get lost when creating a new backup.
+  > For Example: [TachiyomiSY](/forks/TachiyomiSY/) backups have the option to store Read History for titles not in Library.
+  These will **NOT** be restored to original Tachiyomi or other forks.
```

Comment:
Incorrect, these will be restored since they are manga and chapters, they are just not in the library lol

Reply:
🤔 tfw I was living with lies. I'll change it to saved searches